ci/release.sh

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env bash
+
+set -e
+
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+SUBPROJECTS_PROPERTIES="$DIR/../subprojects/gradle.properties"
+ROOT_PROPERTIES="$DIR/../gradle.properties"
+DRY_RUN=0
+
+for arg in "$@"
```

Comment:
Comment on what is happening here (

Reply:
that's cli arguments parsing, some basic way to do it